molecule/cookiecutter/molecule/{{cookiecutter.role_name}}/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/molecule.yml

```diff
@@ -12,6 +16,12 @@ lint:
 platforms:
 {%- if cookiecutter.driver_name == 'azure' %}
   - name: instance
+    vm_size: Standard_A0
```

Comment:
Maybe this is also said in https://github.com/ansible/molecule/pull/1491#issuecomment-434651331 but I think we now need to add a schema to validate this against as in https://github.com/ansible/molecule/blob/master/molecule/model/schema_v2.py. If so, there should also be some unit tests that verify validation works as expected.